uiowa/uiowa
# [Commencement] Collection item bug fix and headline style updates

Fixes #7936.

# How to test
- `ddev blt ds --site=commencement.uiowa.edu && ddev drush @commencement.local uli`
- Visit an event page with ceremony information - like https://commencement.uiowa.ddev.site/ceremony/fall-2018/college-education-commencement-and-teacher-education-program-recognition#arriving-at-hancher-auditorium
- The previous Ceremony information heading should now be Additional Info for Graduates and Guests.
- Check that the style (serif, underlined) of the Livestream and Additional Info headings match the heading style on a venue page like https://commencement.prod.drupal.uiowa.edu/venue/carver-hawkeye-arena. 
- Check that any accordions under Additional Info now render like our other accordions
